spec/acceptance/tests/env_var_existing_accessible.rb

```diff
@@ -0,0 +1,44 @@
+test_name 'PowerShell Module - should see environment variables set outside of session'
```

Comment:
Setting the variable outside of PowerShell is a good idea. Also, I really like the pattern you use for ```apply_manifest_on```. It makes it very readable.